install/kubernetes/helm/istio/charts/ingress-gateway/templates/deployment.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: istio-ingress-gateway
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "istio.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    istio: ingress-gateway
+spec:
+  replicas: {{ .Values.replicaCount }}
```

Comment:
.Values.ingressGateway.replicaCount or .Values.ingress.replicaCount.  I don't have enough context on the gateway functionality to understand if ingress and ingressGateway should go in separate namespaces in values.yaml.